rhel-3tier-iaas/azuredeploy.json

```diff
@@ -0,0 +1,951 @@
+﻿{
+   "$schema":"https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{
+      "adminUsername":{
+         "type":"string",
+         "metadata":{
+            "description":"Username for the Virtual Machines"
+         }
+      },
+      "adminPassword":{
+         "type":"securestring",
+         "minLength":12,
+         "metadata":{
+            "description":"Password for the Virtual Machines(Min Length: 12)"
+         }
+      },
+      "webTierVmCount":{
+         "type":"int",
+         "defaultValue":2,
+         "metadata":{
+            "description":"Number of Web servers to be deployed"
+         }
+      },
+      "appTierVmCount":{
+         "type":"int",
+         "defaultValue":2,
+         "metadata":{
+            "description":"Number of App servers to be deployed"
+         }
+      },
+      "databaseTierVmCount":{
+         "type":"int",
+         "defaultValue":2,
+         "metadata":{
+            "description":"Number of Database servers to be deployed"
+         }
+      }
+   },
+   "variables":{
+      "webTierVmSize":"Standard_D2_v2",

```

Comment:
There are a number of variables that are only used once - can we just use literal property values for those?